engine/src/main/resources/assets/i18n/menu.lang

```diff
@@ -221,6 +221,7 @@
     "listed-servers": "listed-servers",
     "load-delay-warning": "load-delay-warning",
     "load-game": "load-game",
+    "load-game-error": "load-game-error",
```

Comment:
Should the placeholder/dev translation in `menu.lang` indicate that this string is supposed to be a composed message?
```suggestion
    "load-game-error": "load-game-error ({0})",
```

Reply:
_something_ should, whatever the something is that gets shown to the translators.

In the translation systems I've seen previously, that something is the program's "native" language (i.e. English for any project I've worked on);.

I don't know what the keys in this menu.lang map are used for.